src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1218,7 +1222,101 @@ public void addStartDates(Tuple tuple) {
     }
   }
 
-  Future<Void> getUseOverTime(Vertx vertx, RoutingContext ctx) {
+  static void getUseTotalsCsv(JsonObject json, boolean groupByPublicationYear,
+                              boolean periodOfUse, CSVPrinter writer,
+                              String lead) throws IOException {
+    writer.print("Totals - " + lead + " item requests");
+    writer.print(null);
+    writer.print(null);
+    if (periodOfUse) {
+      writer.print(null);
+    }
+    if (groupByPublicationYear) {
+      writer.print(null);
+    }
+    writer.print(null);
+    writer.print(null);
+    writer.print(json.getLong(lead + "ItemRequestsTotal"));
+    Long[] totalItemRequestsPeriod = (Long[]) json.getValue(lead + "ItemRequestsByPeriod");
+    for (int i = 0; i < totalItemRequestsPeriod.length; i++) {
+      writer.print(totalItemRequestsPeriod[i]);
+    }
+    writer.println();
+  }
+
+  static void getUseOverTime2Csv(JsonObject json, boolean groupByPublicationYear,
+                                 boolean periodOfUse, Appendable appendable)
+      throws IOException {
+    CSVPrinter writer = new CSVPrinter(appendable, CSVFormat.EXCEL);
+    writer.print("Title");
+    writer.print("Print ISSN");
+    writer.print("Online ISSN");
+    if (periodOfUse) {
+      writer.print("Period of use");
+    }
+    if (groupByPublicationYear) {
+      writer.print("Year of publication");
+    }
+    writer.print("Access type");
+    writer.print("Metric Type");
+    writer.print("Reporing period total");
+    JsonArray accessCountPeriods = json.getJsonArray("accessCountPeriods");
+    for (int i = 0; i < accessCountPeriods.size(); i++) {
+      // TODO .. If year  prefix with "Published "
+      // MMMM-DD should be converted to "MON-YYYY"
+      writer.print(accessCountPeriods.getString(i));
+    }
+    writer.println();
+
+    writer.print(
```

Comment:
@adamdickmeiss I think we should remove the row with comments on the field -- they are meant as a helper for the sample report but we probably don't need them in the actual data.

Reply:
ok. will do.